core/management/commands/seed.py

```diff
@@ -239,6 +239,7 @@ def create_programs(output=True):
     for program, services in DEFAULT_PROGRAMS.items():
         p = Program()
         p.name = program
+        p.has_queue = random_bool()
```

Comment:
Even though we aren't using the queue right now, which programs have queues should not be random. 
We definitely do not want SEP to have a queue at the moment. 

Create an list of strings const called HAS_QUEUE, with a few program names "STEP" , "LEGAL" and a couple more. then if the program is in the list while seeding, make the has queue flag true